frontend/src/concepts/modelRegistry/utils.ts

```diff
@@ -37,3 +39,27 @@ export const uriToObjectStorageFields = (uri: string): ObjectStorageFields | nul
     return null;
   }
 };
+
+export const getLastCreatedItem = <T extends { createTimeSinceEpoch?: string }>(
+  items?: T[],
+): T | undefined =>
+  items?.toSorted(
+    ({ createTimeSinceEpoch: createTimeA }, { createTimeSinceEpoch: createTimeB }) => {
+      if (!createTimeA || !createTimeB) {
+        return 0;
+      }
+      return Number(createTimeB) - Number(createTimeA);
+    },
+  )[0];
+
+export const filterArchiveVersions = (modelVersions: ModelVersion[]): ModelVersion[] =>
```

Comment:
could combine these to 2 by passing ModelState or even to 1 if you didn't mind doing `models: ModelVersion[] | RegisteredModel[]`, doesn't matter though

Reply:
i see these were just moved from another place anyway 😆

Reply:
Yeah 😆 that's fair.. probably better to factor them out using generics or use `ModelRegistryBase`, but that's tech debt for another time I think (or fine as-is if we never get to it)